ownership-chain/runtime/laos/src/configs/fee/mod.rs

```diff
@@ -0,0 +1,47 @@
+use crate::PhantomData;
+use frame_support::traits::{tokens::currency::Currency, Imbalance, OnUnbalanced};
+
+pub struct DealWithFees<R>(PhantomData<R>);
+
+impl<R> DealWithFees<R>
+where
+	R: pallet_balances::Config + pallet_authorship::Config,
+{
+	// Distributes the specified imbalance to the author of the block
+	fn to_author(imbalance: pallet_balances::NegativeImbalance<R>) {
+		if let Some(author) = <pallet_authorship::Pallet<R>>::author() {
+			<pallet_balances::Pallet<R>>::resolve_creating(&author, imbalance);
+		}
+	}
+}
+
+impl<R> OnUnbalanced<pallet_balances::NegativeImbalance<R>> for DealWithFees<R>
+where
+	R: pallet_balances::Config + pallet_authorship::Config,
+{
+	// Handles unbalanced fees and tips from substrate-based transactions
+	fn on_unbalanceds<B>(
```

Comment:
I think there is no need to implement this. we are sending 100% to author, so only `on_nonzero_unbalanced` is enough. similar to `ToAuthor` in `polkadot` impl

Reply:
very good point